recipes/virtualenv/build.sh

```diff
@@ -0,0 +1,4 @@
+
+#!/bin/bash
+
+$PYTHON setup.py install
```

Comment:
Also if this is using `setuptools` to install, this needs `--single-version-externally-managed --record record.txt`.